game/tests.py

```
@@ -1705,37 +1704,37 @@ def test_stale_game_auto_resignation(self):
-
+        
         response = self.client.post(self.url, HTTP_AUTHORIZATION=f'Bearer {self.secret}')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json()['deleted_games'], 0)
         self.assertEqual(response.json()['resigned_games'], 0)
-
+        
         s1 = SessionStore(session_key=s1.session_key)
         self.assertEqual(s1['game']['game_status'], 'active')
 
     @override_settings(CRON_SECRET='test_secret_123')
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "CRON_SECRET"

(S106)